HolubLife/src/com/holub/rule/Rule.java

```diff
@@ -0,0 +1,9 @@
+package com.holub.rule;
+
+import java.util.List;
+import com.holub.life.Cell;
+
+public abstract class Rule {
+    public abstract boolean willBeAlive(List<Cell> neighbors, boolean amAlive);
```

Comment:
가능하면 Rule이 Cell을 모르는게 더 좋지 않을까요?

```suggestion
    public abstract boolean willBeAlive(int neighborsCount, boolean amAlive);
```

Reply:
달아주신 suggestion에 적극 동감합니다.

Reply:
우선 기획 의도는 생사에 관련한 것들을 전부 Rule에 위임하려고 하였고, 주변에 Active된 셀의 개수만이 아닌 다른 방식으로의 룰도 고려하여 설계를 하였습니다.

Reply:
아 해당 확장까지 고려한다면 지금 설계가 맞는 것 같습니다